components/cli/pkg/commands/list_ingresses.go

```diff
@@ -202,14 +204,14 @@ func displayImageApisTable(imageName string) {
 	}
 
 	if cellImageContent.Kind == "Cell" {
-		displayCellImageApisTable(imageName)
+		displayCellImageApisTable(cli, imageName)
 	} else if cellImageContent.Kind == "Composite" {
-		displayCompositeImageApisTable(imageName)
+		displayCompositeImageApisTable(cli, imageName)
 	}
 }
 
-func displayCompositeImageApisTable(compositeImageContent string) {
-	cell, err := getIngressValues(compositeImageContent)
+func displayCompositeImageApisTable(cli cli.Cli, compositeImageContent string) {
+	cell, err := getIngressValues(cli, compositeImageContent)
 	if err != nil {
 		util.ExitWithErrorMessage("Error occurred while displaying composite image ingress", err)
```

Comment:
Lets return this error and handle from parent level. Please fix in all places.

Reply:
list ingresses command is not refactored yet. Will updated and return the error in the next PR.